test/backend/routes.queries.spec.js

```diff
@@ -114,18 +114,18 @@ describe('Routes:', () => {
                         query: 'SELECT * from ebola_2014 LIMIT 2'
                     };
 
-                    return POST('queries', queryObject);
-                })
-                .then(assertResponseStatus(201))
-                .then(getResponseJson).then(json => {
-                    assert.deepEqual(json, {});
+                    return POST('queries', queryObject)
+                    .then(assertResponseStatus(201))
+                    .then(getResponseJson).then(json2 => {
+                        assert.deepEqual(json2, queryObject);
```

Comment:
This is the only line that needed updating in this test.
Let's revert everything else.